etc/tendrl/snmp.conf.yaml.sample

```diff
@@ -0,0 +1,26 @@
+# V2 end point details
+# Enable the bvelow lines if v2 end point to be used for SNKP notifications
+# Nopte: for different communities make different similar entries below additionally
+# Here default community is public, it is also modifiable
+# V2 is auth less 
+
+# v2_endpoint:
+  # endpoint1:
+    # host_ip: 127.0.0.1
+    # community: public
+  # endpoint2:
+    # host_ip: 127.0.0.1
+    # community: public
+
+# v3_endpoint:
```

Comment:
Add some header comments for v3 endpoints as well and explain the meaning of fields which user needs to add. Just having fields with default values doesn't make enough sense for end user.

Reply:
ok i will add